include/wx/bmpcbox.h

```diff
@@ -97,7 +97,7 @@ class WXDLLIMPEXP_CORE wxBitmapComboBoxBase
 
     void UpdateInternals();
 
-    wxArrayPtrVoid      m_bitmaps;  // Images associated with items
+    wxArrayPtrVoid      m_bitmapbundles;// Images associated with items
```

Comment:
As long as we're changing the name, let's make it a `wxVector<>` too (in a separate commit).